src/scss/includes/components/panel.scss

```diff
@@ -36,13 +38,6 @@
       color: $primary_text;
     }
   }
-
-  &-content {
```

Comment:
I understand you want to get rid of that, but some size limitation is still necessary on desktop so that the content is scrollable. Having the header optional probably makes things more complicated now. Maybe we can find a better way by limiting the `.panel` height itself and use flex for the internal layout between header and content?

Reply:
Done, thanks